### ISSUE TITLE
[Feature Request] Add round corners for windows

### ISSUE THREAD

**Issue by timescam** [pop-os/shell#500]
As title, rounded corners are super sexy. There are patches for some wms. Is there any chance of implementing them in this extension?
Something like [Rounded i3 gaps](https://github.com/resloved/i3) would be nice.

the related [i3 gaps thread](https://github.com/Airblader/i3/issues/167)

**Comment by benediktms** [comment]
I don't think rounded corners are handled by WMs. I would assume that this sort of fucntionallity is usually left to whatever software takes care of window decorations. In the case of Pop_OS that is handled by the GNOME DE which I belive is not very straight forward to edit. You could try to switch to a different DE like KDE plasma (maybe?) which might offer those sort fo customisation options. But not sure how well that plays with the POP_shell and if you are going to all the trouble of tweaking your DE then you might as well use another WM as well 😛

**Comment by jmmaranan** [comment]
It is possible but very tricky using the current GNOME APIs:
![image](https://user-images.githubusercontent.com/348125/90701029-5850a880-e255-11ea-9979-47b89abb9651.png)

**Comment by benediktms** [comment]
Nice, I also found [this](https://www.reddit.com/r/pop_os/comments/icy85d/gnome_my_custom_pop_os/) which might solve the problem :)

**Comment by jmmaranan** [comment]
Yup, would like that more to be CSS and would be curious how the other features of popshell would look like. E.g. active hint, gaps etc

**Comment by nalladev** [comment]
Is this issue still relevant now that COSMIC supports rounded window corners?

**Comment by leviport** [comment]
This repo is for the old Gnome extension, pop-shell, which does not currently round the corners of windows in Gnome.
